reproman/distributions/tests/test_singularity.py

```diff
@@ -21,7 +21,8 @@
 @mark.skipif_no_singularity
 @pytest.mark.xfail(reason="Singularity Hub is down", run=False)
 @pytest.mark.xfail(
-    external_versions["cmd:singularity"] >= '3',
+	# TODO cmd:apptainer version is 1.1.4
+    bool(external_versions["cmd:singularity"]),
```

Comment:
I feel that it is yet another case for `skipif_apptainer` or alike instead of just xfailing it.

Reply:
I went with any singularity version to account for apptainer. We can come back to this and use skipif_apptainer